src/open_producten/producttypen/tests/api/test_producttype.py

```diff
@@ -590,6 +654,42 @@ def test_partial_update_product_type_removing_externe_codes(self):
         self.assertEqual(ExterneCode.objects.count(), 0)
         self.assertEqual(response.data["externe_codes"], externe_codes)
 
+    def test_partial_update_product_type_with_parameter(self):
+        product_type = ProductTypeFactory.create()
+
+        parameters = [{"naam": "doelgroep", "waarde": "inwoners"}]
+        data = {"parameters": parameters}
+        response = self.client.patch(self.detail_path(product_type), data)
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(Parameter.objects.count(), 1)
+        self.assertEqual(response.data["parameters"], parameters)
+
+    def test_partial_update_product_type_with_parameter_replacing_existing(self):
+        product_type = ProductTypeFactory.create()
+        parameter = ParameterFactory.create(product_type=product_type)
+
+        parameters = [{"naam": parameter.naam, "waarde": "test"}]
+        data = {"parameters": parameters}
+        response = self.client.patch(self.detail_path(product_type), data)
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(Parameter.objects.count(), 1)
+        self.assertEqual(response.data["parameters"], parameters)
+
+    def test_partial_update_product_type_removing_parameters(self):
```

Comment:
can you also add a test with patch that updates something else but does not specify parameters? The existing parameters should be kept in that case

Reply:
added